react/localState/mutations.js

```diff
@@ -23,3 +23,9 @@ export const updateItemsSentToServerMutation = gql`
     updateItemsSentToServer @client
   }
 `
+
+export const setIsOpenMutation = gql`
+  mutation setIsOpen($isOpen: Boolean!) {
```

Comment:
shouldn't it be `openMinicart` ?

Reply:
I think that there's no need to create another mutation to close the minicart. It's a toggle (and the mutation is better to be idempotent

Reply:
Okay. So `toggleMinicart`... Because leaving it like this, how is my app supposed to know that the `setIsOpen() @client` mutation is from the minicart?

Reply:
Ok, I'll change it